### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by c-martinez**
Automated testing of endpoints, using virtuoso triple store.

### CODE REVIEW

File: fdp/fdp.py [hunk 1]
```diff
@@ -40,7 +38,7 @@ def initGraph(host, port, dataFile, endpoint):
         g = StoreFAIRGraph(base_uri, endpoint)
     else:
         raise Exception('Unknown data format')
-    data['graph'] = g
+    app.graph = g
```
Comment (CunliangGeng) [inline fdp/fdp.py:41]: Nice refactoring!

File: tests/test_endpoints.py [hunk 2]
```diff
@@ -0,0 +1,304 @@
+import pytest
+
+from fdp.fdp import app, initGraph
+
+
+fdp_post_data = '''
```
Comment (CunliangGeng) [inline tests/test_endpoints.py]: Change these post data to independent `.tll` sample files?

**Review by CunliangGeng — APPROVED** [review]
Thanks Carlos! I'm having a vacation this week and planned to add the tests described in issue #12 next week. Well, it's done now :-)

**Comment by c-martinez** [comment]
Hey @CunliangGeng -- I've moved the post data to separate ttl files like you suggested.

Tests can still be improved. I will leave #12 open and you can decide if these tests are good enough to close the issue :-)

Enjoy your vacation!

**Pull request merged**